content/docs/user-guide/networking/multiplayer/configuration.md

```diff
@@ -1,27 +1,31 @@
 ---
-title: Add the Open 3D Engine Multiplayer Gem to a Project
-description: Add Multiplayer support to an Open 3D Engine project or Gem, including the creation of auto-components.
-linkTitle: Project configuration
+linkTitle: Project Configuration
+title: Adding the Multiplayer Gem to a Project
+description: Learn how to add multiplayer support to an Open 3D Engine project or Gem. Includes instructions for creating a placeholder auto-component.
```

Comment:
We have the new multiplayer template, we should index on setting up a new project using that and then follow the manual steps.

The multiplayer template is the recommended starting point for a new project.

Reply:
I've created a GHI to document use of the template: https://github.com/o3de/o3de.org/issues/2052